docs/selectors.md

```diff
@@ -47,6 +47,11 @@ document
 
 If a selector needs to include `>>` in the body, it should be escaped inside a string to not be confused with chaining separator, e.g. `text="some >> text"`.
 
+### Indexed matches
```

Comment:
Does the index start at 0 or at 1?

Does `page.click('button {2}')` match the second button to fulfill the actionability check? Or always the second button, which is then checked for actionability?

Is `page.click('button {1}')` equivalent to `page.cick('button')`?